src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -199,4 +221,28 @@ internal bool LoadConnections()
         RaisePropertyChanged(nameof(ConnectionSelectionCaptionText));
         return succeeded;
     }
+    
+    private async Task DisplayBindStatusAsync()
+    {
+        var solutionName = await solutionInfoProvider.GetSolutionNameAsync();
+        var isFolderWorkspace = await solutionInfoProvider.IsFolderWorkspaceAsync();
+        SolutionInfo = new SolutionInfoModel(solutionName, isFolderWorkspace ? SolutionType.Folder : SolutionType.Solution);
+        
+        var boundServerProject = connectedModeServices.ConfigurationProvider.GetConfiguration()?.Project;
+        var connection = boundServerProject?.ServerConnection;
+        if (connection == null)
+        {
+            return;
+        }
+        
+        SelectedConnectionInfo = new ConnectionInfo(
```

Comment:
It would be worth extracting the creation of a ConnectionInfo from a ServerConnection into a common place and reuse it, as we do that in multiple places (ServerConnectionsRepositoryAdapter, ServerSelectionViewModel)